firefox-ios/Client/Frontend/Settings/Main/AppSettingsTableViewController.swift

```diff
@@ -473,6 +473,7 @@ class AppSettingsTableViewController: SettingsTableViewController,
             ToggleInactiveTabs(settings: self, settingsDelegate: self),
             ResetContextualHints(settings: self),
             ResetWallpaperOnboardingPage(settings: self, settingsDelegate: self),
+            ResetTermsOfServiceAcceptancePage(settings: self, settingsDelegate: self),
```

Comment:
Should we maybe add this setting only when the feature flag is enabled?

Reply:
Yes, it makes sense. I added this change. Thank you